src/js/components/Settings/AdminShareSettings.vue

```diff
@@ -38,12 +38,12 @@
 				:preserve-search="true"
 				:multiple="true"
 				:loading="isLoading"
-				:placeholder="t('polls', 'Leave empty to disallow for all')"
+				:placeholder="t('polls', 'Leave empty to disallow for all.')"
 				@search-change="loadGroups" />
 		</div>
 
 		<CheckboxRadioSwitch :checked.sync="allAccessLimited" type="switch">
-			{{ t('polls', 'Disallow publishing polls to all users.') }}
```

Comment:
This should definitely be validated to match the rest of the text strings in the settings for this app.